src/main/resources/org/jenkinsci/plugins/workflow/support/steps/build/Messages.properties

```diff
@@ -28,7 +28,9 @@ BuildTriggerStep.no_job_configured=No job configured
 BuildTriggerStep.cannot_find=No such job {0}
 BuildTriggerStep.unsupported=Building a {0} is not supported
 BuildTriggerStepExecution.building_=Building {0}
+BuildTriggerStepExecution.scheduling_=Scheduling {0}
```

Comment:
BTW the trailing `_` in the original key was probably a mistake; no need to copy it.